### PR TITLE
commit challenge.rb

### PR DESCRIPTION
**Pull request by ryoconnell**

### CODE REVIEW

File: challenge.rb [hunk 1]
```diff
@@ -6,7 +6,15 @@
 # and should return either `true` or `false` based on whether `age` is less than `minimum_age`.
 
 def is_of_age? age, minimum_age
-  # Replace this comment with your code.
+  minimum_age = ()
```
Comment (MattB-GA-Boston) [inline challenge.rb]: ?

What is this? Where did you see it?

File: challenge.rb [hunk 2]
```diff
@@ -6,7 +6,15 @@
...
+  age = ()
+  print "What is your age?"
+  age = gets.chomp
```
Comment (MattB-GA-Boston) [inline challenge.rb]: No `gets` or `puts` needed here - `age` and `minimum_age` are being passed in as arguments.

File: challenge.rb [hunk 3]
```diff
@@ -6,7 +6,15 @@
...
+  if age > minimum_age
+    return "True"
```
Comment (MattB-GA-Boston) [inline challenge.rb]: "True" is a string. `true` is the boolean.

File: challenge.rb [hunk 4]
```diff
@@ -6,7 +6,15 @@
...
+  else
+    return "Fale"
+
```
Comment (MattB-GA-Boston) [inline challenge.rb]: Your logic isn't far off, but your syntax is out in left field.

File: challenge.rb [hunk 5]
```diff
@@ -22,7 +30,11 @@ def is_of_age? age, minimum_age
 # Remember: `p`, `print`, and `puts` don't return their arguments.
 
 def okay_to_drink? age, drinking_age
-  # Replace this comment with your code.
+  drinking_age = ()
+  age = ()
+  if is_of_age return "True"
```
Comment (MattB-GA-Boston) [inline challenge.rb]: Close on this one, but not quite.
```
if is_of_age?
  "Come on in." #alternatively, `return "Come on in."`
```

File: challenge.rb [hunk 6]
```diff
@@ -22,7 +30,11 @@ def is_of_age? age, minimum_age
...
+    print "Come on in"
+  else return "I'm sorry, #{age} isn't old enough to drink.  The minimum age is #{drinking_age}"
```
Comment (MattB-GA-Boston) [inline challenge.rb]: The second half of this is right, but you're missing the final `end`!

File: challenge.rb [hunk 7]
```diff
@@ -45,7 +57,17 @@ def okay_to_drink? age, drinking_age
 #   "Fix the AC whenever you have the chance. It's cool."
 
 def ac_monitor current_temp, ac_working, desired_temp
-  # Replace this comment with your code.
+  if ac_working = true && current_temp > desired_temp
+    return "Turn on the AC."
+
```
Comment (MattB-GA-Boston) [inline challenge.rb]: In Ruby, any time you have an `if... elsif... else`, you need an `end`. Anytime you define a method, you need an `end`.

File: challenge.rb [hunk 8]
```diff
@@ -45,7 +57,17 @@ def okay_to_drink? age, drinking_age
...
+  if ac_working = true && current_temp <= desired_temp
+    return "Just right!"
+
+  if ac_working = false && current_temp > desired_temp
+    return "Fix the AC now! It's hot!"
+  if ac_wokring = false && current_temp <= desired_temp
+    return "Fix the AC whenever you have the changce.  It's cool."
+  end
```
Comment (MattB-GA-Boston) [inline challenge.rb]: Apart from the missing `end`s and a few typos, this is correct.

File: challenge.rb [hunk 9]
```diff
@@ -57,7 +79,15 @@ def ac_monitor current_temp, ac_working, desired_temp
...
-# Replace this comment with your code.
+def fizz_buzz_calculator num
+  if num % 3 == 0
+    return "fizz"
+  if num % 5 == 0
+    return "buzz"
+  if num % 3 == 0 && num % 5 == 0
+    return "fizzbuzz"
+  else
+    return "num"
```
Comment (MattB-GA-Boston) [inline challenge.rb]: Spot on!! Just watch those missing `end`s. You need one after the `else` and one to end the method.

**Comment by MattB-GA-Boston** [comment]
2 / 5 (-0.75 on Q1, -0.5 on Q2, -0.5 on Q3, -0.25 on Q4, -1 on Q5)

This definitely could have gone better, but most of the mistakes stemmed from forgetting to `end` things when you should. Try submitting this one again sometime this week or coming weekend; I'm confident that you'll do much better.

**Comment by ryoconnell** [comment]
Sounds good.  I feel better about it after today.

**Pull request closed without merging**